akka-bbb-apps/src/main/scala/org/bigbluebutton/core/db/PresPageDAO.scala

```diff
@@ -47,4 +61,51 @@ object PresPageDAO {
         case Failure(e)            => DatabaseConnection.logger.debug(s"Error updating current on PresPage: $e")
       }
   }
+
+  def resizeAndMovePage(presentation: PresentationPage, presentationId: String) = {
+    DatabaseConnection.db.run(
+      sqlu"""UPDATE pres_page SET
+            "xOffset" = ${presentation.xOffset},
+            "yOffset" = ${presentation.yOffset},
+            "widthRatio" = ${presentation.widthRatio},
+            "heightRatio" = ${presentation.heightRatio}
+            WHERE "presentationId" = $presentationId"""
```

Comment:
It is updating all rows of the presentation.
Whereas it should update only the specific `slideId`.